pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -233,7 +233,7 @@ class VmOverviewTabLibvirt extends React.Component {
                         );
                     } else {
                         firmwareLinkWrapper = (
-                            <Tooltip id='firmware-edit-disabled-on-transient' content={_("Transient VMs don't support editting firmware configuration")}>
+                            <Tooltip id='firmware-edit-disabled-on-transient' content={_("Transient VMs don't support editing firmware configuration")}>
```

Comment:
This is a good change of course, but changes to translated strings should also be done in po/*.po, to avoid breaking all non-English languages. I. e. just the same s/editting/editing/ in all po/*.po.

Reply:
Done. Thanks! Caught another one for "dependant" to "dependent" when running `codespell` on `uk.po`.